### ISSUE TITLE
relevant line detection

### ISSUE THREAD

**Issue by hdgarrood** [colszowka/simplecov#148]
Firstly, this gem is sweet :) thanks very much!

There's a line in a gem I'm developing, which I think should be relevant, but simplecov (or perhaps, Coverage) disagrees. The problem occurs with hdgarrood/vdf_parser@820f803 . After doing `bundle exec rake coverage` I would expect line 26 of lib/vdf_parser/vdf_tokenizer.rb to be counted as relevant, and highlighted green. Instead, it is not highlighted, which I assume means it was counted as irrelevant. see http://imagebin.org/220120

I also tried to do the same with Coverage, by issuing:

```
$ bundle exec irb
>> require 'coverage.so'
=> true
>> Coverage.start
=> nil
>> require 'test/unit'
=> true
>> Dir.glob('test/tc_*.rb').each do |f|
?> require "./#{f}"
>> end
=> ["test/tc_abstract_tokenizer.rb", "test/tc_vdf_tokenizer.rb"]
>> p Coverage.result["/path/to/vdf_parser/lib/vdf_parser/vdf_tokenizer.rb"]
[1, nil, 1, 1, 1, nil, 1, 0, nil, nil, 1, nil, nil, nil, nil, nil, nil, nil, nil, nil, 0, nil, nil, 1, nil, nil, nil, nil, 1, nil, 0, 0, nil, nil, 1, 0, 0, nil, nil, nil, nil]
```

but now I am even more confused, because the lines which simplecov says have been hit twice have a 0 from Coverage.

In summary:
- Am I incorrect in thinking that line 26 of lib/vdf_parser/vdf_tokenizer.rb is relevant?
- Do you know why Coverage.result doesn't seem to match with simplecov? (or can you point me to some documentation which explains how to interpret Coverage.result?)
